main/src/cgeo/geocaching/filters/gui/GeocacheFilterActivity.java

```diff
@@ -184,27 +183,24 @@ protected void onRestoreInstanceState(@NonNull final Bundle savedInstanceState)
 
     @Override
     public boolean onCreateOptionsMenu(final Menu menu) {
-        super.onCreateOptionsMenu(menu);
```

Comment:
Call to "super" method should stay imho - unless there's a specific reason to omit it?

Reply:
The super call is not really needed. The only purpose would be if we would have a standard menu defined for all activities and want to extend it. But otherwise, it has no advantages. That's why there's also no lint error like for other "missing super calls"... In other activities we often  don't have super calls for it as well. 

But if you insist I can add it again, as it doesn't hurt either ;-)

Reply:
No I am not insisting.

But unfortunately the PR has merge conflicts now